Readme.md

```diff
@@ -273,6 +273,10 @@ Implements the `Date` object but using the clock to provide the correct time.
 
 Implements the `now` method of the [`Performance`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) object but using the clock to provide the correct time. Only available in environments that support the Performance object (browsers mostly).
 
+### `lolex.lolexFromGlobal`
```

Comment:
Any reason this has `lolex` in the name (rather than just `fromGlobal` or `withGlobal`?

Reply:
mostly as it kinda made sense in my head to do `const lolex = lolexFromGlobal(myGlobal)`. But I'm totally fine with another name. Any preferences?

Reply:
I'd like `withGlobal` or `fromGlobal`. Looks better IMHO `lolex.withGlobal(myJsDom)`

If people do static imports and think the contextless names are confusing they can name it whatever they like anyway:
`import {withGlobal as lolexFromGlobal} from 'lolex'`.